src/Nri/Ui/Text/V1.elm

```diff
@@ -150,6 +155,42 @@ classString classes =
     String.join " " (List.map (identifierToString namespace) classes)
 
 
+{-| User-generated text.
+-}
+ugMediumBody : List (Html.Styled.Html msg) -> Html.Styled.Html msg
+ugMediumBody =
+    Html.Styled.p
+        [ css
+            [ quizFont
+            , fontSize (px 18)
+            , lineHeight (px 30)
+            , whiteSpace preLine
+            , color gray20
+            , margin4 (px 10) (px 0) (px 0) (px 0)
+            , firstChild [ margin zero ]
+            , firstOfType [ margin zero ]
+            ]
+        ]
+
+
+{-| User-generated text.
+-}
+ugSmallBody : List (Html.Styled.Html msg) -> Html.Styled.Html msg
+ugSmallBody =
+    Html.Styled.p
+        [ css
+            [ quizFont
+            , fontSize (px 16)
+            , lineHeight (px 25)
+            , whiteSpace preLine
+            , color gray20
+            , margin4 (px 7) (px 0) (px 0) (px 0)
+            , firstChild [ margin zero ]
+            , firstOfType [ margin zero ]
```

Comment:
@bendansby is this what you wanted? first-child or first-of-type?

Reply:
Whoops, guess I asked for a property that doesn't exist 😅 

That looks good, though!

Reply:
But one that should exist! I would like for first-of-class to be a thing too